components/reusable-components/navigation/navigation.module.scss

```diff
@@ -0,0 +1,126 @@
+@mixin respond-to($size) {
+  @if $size == 'small' {
+    @media (max-width: 768px) {
+      @content;
+    }
+  } @else if $size == 'medium' {
+    @media (min-width: 769px) and (max-width: 1024px) {
+      @content;
+    }
+  } @else if $size == 'large' {
+    @media (min-width: 1025px) and (max-width: 1440px) {
+      @content;
+    }
+  } @else if $size == 'extra-large' {
+    @media (min-width: 1441px) {
+      @content;
+    }
+  }
+}
+
+.navbar {
+  background-color: var(--bg-color);
+}
+
+.NavigationLogo {
+  color: var(--text-color);
+  margin-left: 0.55em;
+  width: 6.3em;
+  height: 1.22em;
+  top: 1.86em;
+  left: 2.36em;
+}
+
+.menuElements {
+  display: flex;
+  color: var(--text-color);
+  position: relative;
+  margin-left: 4.16em;
+  left: 10.41em;
+}
+
+.menuElementsNav {
+  display: flex;
+  list-style: none;
+  padding: 10px;
+}
+
+.navContainer {
+  display: flex;
+  justify-content: space-between;
+}
+
+.leftContent {
+  display: flex;
+  align-items: center;
+}
+
+.buttonsContent {
+  display: flex;
+  align-items: center;
+}
+
+.largeNavbar {
+  background-color: var(--bg-color);
+  padding: 40px;
+}
+
+@include respond-to('small') {
+  .navbarBrand {
+    margin-left: 0.25em;
+    width: 4em;
+    height: 1em;
+    top: 1.5em;
+    left: 1.5em;
+  }
+
+  .navLink1 {
```

Comment:
@Mikivitkovski as I mentioned before, we can improve the naming for this class, or use a child selector instead.